lib/auth/cognito_chooser.rb

```diff
@@ -24,20 +27,26 @@ def aws_secret_key
     Rails.configuration.cognito_aws_secret_access_key
   end
 
+  def region
```

Comment:
this is still unused, see my comment on the resolved comment... nothing is calling this method.. other than this, it looks good to be merged!

Reply:
Removed... I made the mistake of thinking I was using it for configuring cognito on line 47.